### ISSUE TITLE
Tuto on Tween

### ISSUE THREAD

**Issue by Patlol** [godotengine/godot-docs#3054]
If you would like a tutorial on tween with examples on the different parameters TransitionType and EaseType could be very interesting ...

**Comment by hsandt** [comment]
Yes, I was confused by how `set_ease` doc example says to just use `tween.set_ease(Tween.EASE_IN)` but it changes nothing without `set_trans` combined with it, since Linear is just Linear. I had to check this animated cheatsheet made by a user to remind myself of it: https://www.reddit.com/r/godot/comments/14gt180/all_possible_tweening_transition_types_and_easing/

So a full example explaining how it's important to set TransitionType would really help beginners not getting stuck on that (example: https://www.reddit.com/r/godot/comments/13ste37/tweenset_easetweenease_in_out_isnt_working_am_i/)
